src/js/sites/GitHub/api.js

```diff
@@ -18,14 +18,14 @@ export function bypassCSPForImages(images = []) {
     )).join('\n\n');
 
     return getMarkdownPreview(markdown).then(res => {
-        const $res = $(res);
-        return Array.from($res.filter('p')).map(p => {
-            const imgs = $(p).find('img');
-            return {
-                uri: imgs.get(0).src,
-                name: imgs.get(0).alt,
-                downsizedUri: imgs.get(1).src
-            };
-        });
+      const $res = $(res);
```

Comment:
Would you mind fixing the indentation here to 4 spaces to match the rest of the file? 😃